pkgs/top-level/all-packages.nix

```diff
@@ -27763,9 +27763,7 @@ with pkgs;
 
   wpa_supplicant = callPackage ../os-specific/linux/wpa_supplicant { };
 
-  wpa_supplicant_ro_ssids = wpa_supplicant.override {
-    readOnlyModeSSIDs = true;
-  };
+  wpa_supplicant_ro_ssids = wpa_supplicant;
```

Comment:
If this is now integrated into the default package, I think it would be good to show a warning here to remind users that they can switch to the main package.

Reply:
I added a warning. Is lib.trivial.warn the right fn to call here?

Reply:
Yes that should be fine. Thanks

Reply:
Yes that should be fine, thanks!

Reply:
This should be moved into `pkgs/top-level/aliases.nix`.

Reply:
Thanks & fixed.